src/components/preference-group/preference-group.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  selector: 'app-preference-group',
+  templateUrl: './preference-group.component.html',
+  styleUrls: ['./preference-group.component.scss'],
+})
+export class PreferenceGroupComponent {
+  @Input() title: string;
+  @Input() subject: BehaviorSubject<boolean>;
```

Comment:
How about preference$ or some other name with a bit more of an indication of what value the user can expect?

Reply:
sounds good! one thing i'm worried about is is it conventionally okay to have `$` in the name of an input property?
```html
<app-preference-group [preference$]="...">
```